packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -161,13 +161,15 @@ export class ChoiceGroup extends React.Component<IChoiceGroupProps, IChoiceGroup
   }
 
   private _onChange(option: IChoiceGroupOption, evt: React.FormEvent<HTMLInputElement>) {
-    let { onChanged } = this.props;
+    let { onChanged, onChange } = this.props;
 
     this.setState({
       keyChecked: option.key
     });
 
-    if (onChanged) {
+    if (onChange) {
+      onChange(evt, option);
+    } else if (onChanged) {
       onChanged(option);
```

Comment:
General question about deprecation: should we log a console warning if this is used? Otherwise existing consumers might not know that this is being deprecated (not everyone reads release notes...) and their sites will just break once removed.

Ideally this would only be done in a non-minified/debug build, not sure how that would work when people bundle themselves.